### PR TITLE
hide strange Everyone and Disabled group numbers with LDAP

### LINKED ISSUE #17368: Everyone and disabled count with LDAP
When LDAP is enabled, the counts for everyone is always 1 and for disabled users always -1. That'S due to disabled counting, but those should be hidden.

### PR DESCRIPTION
**Pull request by blizzz**
fixes #17368

Before:

Screenshot_20191007_182750

Now:

Screenshot_20191007_182925

### CODE REVIEW

**Review by juliusknorr — APPROVED** [review]

**Comment by juliushaertl** [comment]
Bundles need to be updated as well

File: apps/settings/src/views/Users.vue [hunk 1]
```diff
@@ -283,6 +283,10 @@ export default {
 					|| disabledGroup.utils.counter === -1)				// add disabled if ldap enabled
 				) {
 					groups.unshift(disabledGroup)
+					if (disabledGroup.utils.counter === -1) {
+						// hides the counter instead of showing -1
+						disabledGroup.utils.counter = 0
```
Comment (skjnldsv) [inline apps/settings/src/views/Users.vue]: I would even remove it directly? :thinking:  like `delete disabledGroup.utils.counter` :)

Comment (blizzz) [reply apps/settings/src/views/Users.vue]: works too

File: apps/settings/src/views/Users.vue [hunk 2]
```diff
@@ -283,6 +283,10 @@ export default {
 					|| disabledGroup.utils.counter === -1)				// add disabled if ldap enabled
 				) {
 					groups.unshift(disabledGroup)
+					if (disabledGroup.utils.counter === -1) {
+						// hides the counter instead of showing -1
+						delete disabledGroup.utils.counter
```
Comment (skjnldsv) [inline apps/settings/src/views/Users.vue]: Ah wait, I'm an idiot.
Did not saw the line above. If you do this, then the next call will remove the disable entry in the menu :see_no_evil: 

We should definitely find another way :grimacing:

Comment (blizzz) [reply apps/settings/src/views/Users.vue]: Nope, at least it was not removed for me locally.

Comment (skjnldsv) [reply apps/settings/src/views/Users.vue]: I assumed (wrongly? :thinking: :grin: )

Comment (blizzz) [reply apps/settings/src/views/Users.vue]: As said, it works for me both ways.

**Comment by skjnldsv** [comment]
> /compile /

this is not a rebase bot :)

**Comment by blizzz** [comment]
it's a bit cumbersome for resources only being built

**Comment by blizzz** [comment]
rebased again…

**Review by skjnldsv — APPROVED** [review]

**Comment by blizzz** [comment]
Ah, I have an addition: group numbers are also increased or decreased when a user is added or removed from a group.  Commit comming soon.

**Pull request merged**